usagov/usagov-2021
# USAGOV-1788: Add Docker Hub credentials to deploy-reporter

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1788

## Description

Add the authentication-enabled form of `cf push` to the deploy-reporter script

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

Look it over! I made similar changes to deploy-cms, deploy-waf, and deploy-www, and it appears that deploy-cfevents will probably need them too. 

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
